devel/libecl/include/ert/ecl/ecl_sum.h

```diff
@@ -79,6 +79,8 @@ typedef struct ecl_sum_struct       ecl_sum_type;
   const char *     ecl_sum_iget_wgname( const ecl_sum_type * sum , int param_index );
   const char *     ecl_sum_iget_keyword( const ecl_sum_type * sum , int param_index );
   int              ecl_sum_get_data_length( const ecl_sum_type * ecl_sum );
+  void             ecl_sum_scale_vector( ecl_sum_type * ecl_sum, int index, double scalar );
```

Comment:
`scale_factor` or `factor` would have been a better name than `scalar` - the fact that it scalar can actually just be read from the declaration, but says nothing of how it will be used. Anyway - just a minor minor detail; merging.